providers/flagd/README.md

```diff
@@ -116,6 +116,7 @@ Given below are the supported configurations:
 | streamDeadlineMs      | FLAGD_STREAM_DEADLINE_MS       | int                      | 600000    | rpc & in-process        |
 | keepAliveTime         | FLAGD_KEEP_ALIVE_TIME_MS       | long                     | 0         | rpc & in-process        |
 | selector              | FLAGD_SOURCE_SELECTOR          | String                   | null      | in-process              |
+| providerId            | FLAGD_SOURCE_PROVIDER_ID       | String                   | null      | in-process              |
```

Comment:
Should we use `providerId` or `providerID`? In Go, the later was used and that's what's defined [here](https://github.com/open-feature/flagd/pull/1567).

Reply:
depending on the decision, we might want to add it [here](https://github.com/open-feature/flagd-testbed/blob/main/gherkin/config.feature) for easier testing throughout out all of our sdks

Reply:
Oops, fixed to be consistent with Go.

Reply:
We've use camel case for "id" elsewhere in general (like flagSetId). I think I'd prefer that @beeme1mr

Reply:
I'm fine with that. I honestly prefer camel case here but I wanted to call out the inconstancy. As long as the environment variable is the same, it's probably better to follow the languages best practices than force a particular style.